Applications/WebPortal/WebApp/library/source/pages/account_directory_page/account_directory_page/account_directory_controller.tsx

```diff
@@ -56,7 +56,8 @@ export class AccountDirectoryController extends
     return <AccountDirectoryPage
       displaySize={this.props.displaySize}
       roles={this.props.roles}
-      groups={this.props.model.groups}
+      groups={this.props.model.groups.sort(
```

Comment:
This is going to be way too slow. Avoid computing things on every render, instead compute the value once and then reuse it.